tests/functional/test_contract_instance.py

```diff
@@ -38,6 +38,9 @@ def test_init_specify_contract_type(
 def test_contract_calls(owner, contract_instance):
     contract_instance.setNumber(2, sender=owner)
     assert contract_instance.myNumber() == 2
+    contract_instance.invoke_transaction("setNumber", 3, sender=owner)
```

Comment:
This part should probably in a separate test, like `test_invoke_transaction()` or something.
Also, let's assert something:

```suggestion
    receipt = contract_instance.invoke_transaction("setNumber", 3, sender=owner)
    assert not receipt.failed
```